usecases/modulecomponents/batch/tokenization.go

```diff
@@ -0,0 +1,68 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package batch
+
+import (
+	"context"
+
+	"github.com/weaviate/weaviate/usecases/modulecomponents/settings"
+
+	"github.com/weaviate/weaviate/entities/moduletools"
+
+	"github.com/weaviate/tiktoken-go"
+	"github.com/weaviate/weaviate/entities/models"
+	"github.com/weaviate/weaviate/modules/text2vec-openai/clients"
+	objectsvectorizer "github.com/weaviate/weaviate/usecases/modulecomponents/vectorizer"
+)
+
+type TokenizerFuncType func(ctx context.Context, objects []*models.Object, skipObject []bool, cfg moduletools.ClassConfig, objectVectorizer *objectsvectorizer.ObjectVectorizer) ([]string, []int, bool, error)
+
+func ReturnBatchTokenizer(multiplier float32, moduleName string) TokenizerFuncType {
```

Comment:
would be great to expand the method with atl names
```
func ReturnBatchTokenizer(multiplier float32, moduleName string, moduleAltNames ...string) TokenizerFuncType
```

and then use this in line 34 like so:
```
icheck := settings.NewBaseClassSettingsWithAltNames(cfg, moduleName, moduleAltNames),
```

and in `text2vec-google` module you would have to also pass altNames